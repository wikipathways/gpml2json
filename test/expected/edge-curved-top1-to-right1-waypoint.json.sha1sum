abde35a24437f4f63f4f5487619fa1925e3e1bdf ?-
