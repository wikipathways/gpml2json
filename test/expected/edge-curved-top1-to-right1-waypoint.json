{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["c0fac","fe12c","fc22c"],"drawAs":"rect","gpmlElementName":"Pathway","height":825.09187756604,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":1047.7865523249097,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/584010709fed216f36e4b0503b791811112abf11b29952fc38c4e40333793537"},"entityMap":{"fe12c":{"id":"fe12c","textContent":"GeneProduct","wpType":"GeneProduct","x":173.9937029287637,"y":94.91908337772578,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32972,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"fc22c":{"id":"fc22c","textContent":"0004","wpType":"GeneProduct","x":23.354108824975164,"y":75.64749377680056,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32973,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"c0fac":{"id":"c0fac","drawAs":"CurvedLine","borderWidth":1,"zIndex":32971,"points":[{"isAttachedTo":"fc22c","attachmentDisplay":{"position":[0.5,0],"offset":[0,0]},"x":63.354108824975164,"y":75.64749377680056,"orientation":[0,-1]},{"x":182.6889787043376,"y":66.33070321840864},{"x":302.0238485837,"y":85.6248932980672},{"isAttachedTo":"fe12c","attachmentDisplay":{"position":[1,0.5],"offset":[0,0]},"x":253.9937029287637,"y":104.91908337772578,"orientation":[-1,0]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerStart":"TBar","markerEnd":"Arrow","isAttachedTo":["fc22c","fe12c"]}}}
